docs/installation/presto-configuration.rst

```diff
@@ -17,66 +18,83 @@ memory (say, 120GB/node), you may want to allocate more memory to Presto for bet
 In order to update the max memory value to 60 GB per node:
 
 1. Change the line in ``/etc/opt/prestoadmin/coordinator/jvm.config`` and
-``/etc/opt/prestoadmin/workers/jvm.config`` that says ``-Xmx16G`` to ``-Xmx60G``.
+   ``/etc/opt/prestoadmin/workers/jvm.config`` that says ``-Xmx16G`` to ``-Xmx60G``.
 
 2. Change the following lines in ``/etc/opt/prestoadmin/coordinator/config.properties``
-and ``/etc/opt/prestoadmin/workers/config.properties``: ::
+   and ``/etc/opt/prestoadmin/workers/config.properties``: ::
 
-    query.max-memory-per-node=8GB
-    query.max-memory=50GB
+        query.max-memory-per-node=8GB
+        query.max-memory=50GB
 
 
-to ::
+   to ::
 
-    query.max-memory-per-node=30GB
-    query.max-memory=<30GB * number of nodes>
+        query.max-memory-per-node=30GB
+        query.max-memory=<30GB * number of nodes>
 
 
-We recommend setting ``query.max-memory-per-node`` to half of the JVM config max memory, though if your workload is highly concurrent, you may want
-to use a lower value for ``query.max-memory-per-node``.
+   We recommend setting ``query.max-memory-per-node`` to half of the JVM config max memory, though if your workload is highly concurrent, you may want
```

Comment:
You may mention that by default it is 10% of the heap space allocated to Presto JVM